platform/broadcom/saibcm-modules/sdklt/linux/bde/ngbde_dma.c

```diff
@@ -224,7 +227,9 @@ ngbde_dmamem_free(ngbde_dmamem_t *dmamem)
         }
         dma_free_coherent(dmamem->dev, dmamem->size,
                           dmamem->vaddr, dmamem->paddr);
-        memset(dmamem, 0, sizeof(*dmamem));
+        #ifdef __STDC_LIB_EXT1__
```

Comment:
>__STDC_LIB_EXT1__ [](http://example.com/codeflow?start=15&length=17)

If the macro is not defined, you need to fallback to original function.

Reply:
Make senses. Updated fallback function.